### PR TITLE
Add support for abc.com.au

### LINKED ISSUE #2436: Add Triple J to list of supported websites
Hi,

I recently started using Last.fm and in order to start scrobbling as much music as possible, I added the Web Scrobbler extension to my browser. Because I listen to internet radio quite a lot, I would really appreciate it if you could add support for Triple J, specifically the Triple J Unearthed Jukebox website.

Thanks!

### PR DESCRIPTION
**Pull request by inverse**
Resolves #2436

Supports all 3 players on the page.

### CODE REVIEW

**Comment by alexesprit** [comment]
Could you not use jQuery?

File: src/core/connectors.js [hunk 1]
```diff
@@ -1497,6 +1497,13 @@ const connectors = [{
 	],
 	js: 'connectors/napster.js',
 	id: 'napster',
+}, {
+	label: 'triplej',
+	matches: [
+		'*://www.abc.net.au/triplej/listen-live/*',
```
Comment (alexesprit) [inline src/core/connectors.js:1503]: We could make a more generic connector that covers all radio stations from ABC.

**Comment by inverse** [comment]
When did you deprecate support and why?

**Comment by alexesprit** [comment]
> When did you deprecate support and why?

https://github.com/web-scrobbler/web-scrobbler/wiki/Connector-API-changes#v2310

It was mostly used along with Bootstrap, but jQuery will be removed in Bootstrap 5. So, there's no sense to keep a dependency, when we can do everything with DOM API.

Using jQuery in connectors to get a text from an element is overkill.

File: src/connectors/triplej.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+Connector.playerSelector = '#main-content';
+
+Connector.getArtist = () => {
+	return $('.playingNow div[class*="songArtist"]').text() || $('.view-liveMusicNow .artist').text();
```
Comment (alexesprit) [inline src/connectors/triplej.js:6]: This can be replaced by an array of selectors.

**Comment by inverse** [comment]
Why reducing the number of 3rd party dependencies can be a good thing I think leveraging something as well known as jQuery aids the barrier to entry for people to contribute. Although I know vanilla JS has come on a long way.

File: src/core/connectors.js (hunk 1 above)
Comment (inverse) [reply src/core/connectors.js:1503]: Which other ones did you find?

Comment (inverse) [reply src/core/connectors.js:1503]: Ahhh i see what you mean. I was accessing them all via `/triplej/` but you can access them directly

File: src/connectors/triplej.js (hunk 2 above)
Comment (inverse) [reply src/connectors/triplej.js]: Got an example where we do similar?

Comment (inverse) [reply src/connectors/triplej.js]: Figured

**Comment by inverse** [comment]
What's wierd is the triplej player now for me isnt' expanding with track info

image

The other two are just fine - but they have different selectors

File: src/connectors/abcnetau.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Connector.playerSelector = '#main-content';
+
+Connector.artistSelector = '.playingNow div[class*="songArtist"], .view-liveMusicNow .artist';
+
+Connector.trackSelector = '.playingNow h3, .view-liveMusicNow .title';
+
+Connector.albumSelector = '.playingNow div[class*="songRelease"], .view-liveMusicNow .release';
+
+Connector.isPlaying = () => {
+	const $playerElement = document.querySelector('#jwplayerDiv, #radio-player4-player');
```
Comment (alexesprit) [inline src/connectors/abcnetau.js:12]: You can use `Util.hasElementClass` instead.

Comment (inverse) [reply src/connectors/abcnetau.js]: Updated

**Pull request merged**